### ISSUE TITLE
Can I Publish The app on Play Store

### ISSUE THREAD

**Issue by itzsachin321** [AzizStark/Quiva#15]
Please tell me can I publish on play store and tell me you don't give me copyright

**Comment by AzizStark** [comment]
Yes you can, but **the source code of the work must be made available under the same license and you should credit my name**.

**Comment by itzsachin321** [comment]
Thank,
I will give name in app discrition

**Comment by itzsachin321** [comment]
Thanks,
I will give your name in app and discription

**Comment by BradKML** [comment]
Oof https://github.com/AzizStark/Quiva/issues/11
